content/items/rggen.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "RgGen"
+description: "Code generator for configuration and status registers"
+authors:
+  - Taichi Ishitani
+links:
+  gh?: "rggen/rggen"
```

Comment:
You might want to point to the org, instead of the repo. So, `gh: rggen`.

Reply:
This repository contains all information about RgGen.
Therefore, I'd like to keep this field as is.